consistent-sampling/build.gradle.kts

```diff
@@ -0,0 +1,13 @@
+plugins {
+  id("otel.java-conventions")
+  id("otel.publish-conventions")
+}
+
+description = "Sampler and exporter implementations for consistent sampling"
+
+dependencies {
+  api("io.opentelemetry:opentelemetry-sdk")
+  testImplementation("com.google.guava:guava:31.0.1-jre")
```

Comment:
```suggestion
  testImplementation("com.google.guava:guava")
```

Reply:
Though given it'd be very simple to replace the one usage with `HashMap` would recommend doing that instead

Reply:
fixed in 6b4ba10eb16ba8392945f1e59b852e9b4f6be637